tests/components/tibber/test_statistics.py

```diff
@@ -71,3 +74,30 @@ def _get_homes():
 
         _sum += _CONSUMPTION_DATA_1[k]["consumption"]
         assert stat["sum"] == _sum
+
+    # Validate cost
+    statistic_id = "tibber:energy_totalcost_home_id"
+
+    stats = await hass.async_add_executor_job(
+        statistics_during_period,
+        hass,
+        dt_util.parse_datetime(_CONSUMPTION_DATA_1[0]["from"]),
+        None,
+        [statistic_id],
+        "hour",
+        True,
+    )
+
+    assert len(stats) == 1
+    assert len(stats[statistic_id]) == 3
+    _sum = 0
+    for k, stat in enumerate(stats[statistic_id]):
+        assert stat["start"] == dt_util.parse_datetime(_CONSUMPTION_DATA_1[k]["from"])
+        assert stat["state"] == _CONSUMPTION_DATA_1[k]["totalCost"]
+        assert stat["mean"] is None
```

Comment:
Is there a way to test this with `mean`, `min`,  `max`, etc populated?

Reply:
No. 
There ar only hourly data, so no mean, min or max are available